larpandoracontent/LArCheating/CheatingNeutrinoIdTool.cc

```diff
@@ -115,7 +40,7 @@ void CheatingNeutrinoIdTool::SelectOutputPfos(const pandora::Algorithm *const /*
             LArPfoHelper::GetAllDownstreamPfos(pNeutrinoPfo, downstreamPfos);
 
             float thisNeutrinoWeight(0.f), thisTotalWeight(0.f);
-            CheatingNeutrinoIdTool::GetNeutrinoWeight(&downstreamPfos, false, thisNeutrinoWeight, thisTotalWeight);
+            CheatingParticleIdBaseTool::GetTargetParticleWeight(&downstreamPfos, false, thisNeutrinoWeight, thisTotalWeight, LArMCParticleHelper::IsNeutrino);
             neutrinoWeight += thisNeutrinoWeight;
         }
 
```

Comment:
Note, for reference: not dividing by total weight here (in neutrino id version), just using numerator.